tests/ipsecondary_test.py

```diff
@@ -232,6 +232,36 @@ def test_primary_timeout_after_device_is_registered(uptane_repo, secondary, aktu
     return aktualizr.get_current_primary_image_info() == primary_image_hash
 
 
+@with_uptane_backend()
+@with_secondary(start=False)
+@with_secondary(start=False, arg_name='secondary2')
+@with_aktualizr(start=False, output_logs=True)
+def test_primary_multiple_secondaries(uptane_repo, secondary, secondary2, aktualizr, **kwargs):
+    '''Test Aktualizr with multiple ip secondaries'''
+
+    with aktualizr, secondary, secondary2:
+        aktualizr.wait_for_completion()
```

Comment:
This is all happy-path, with everything more or less starting simultaneously, right? Can we extended this for the cases when one or more Secondaries don't come up immediately?